src/main/java/frc/robot/subsystems/swerve/modules/ModuleConstants.java

```diff
@@ -1,20 +1,32 @@
 package frc.robot.subsystems.swerve.modules;
 
 import edu.wpi.first.math.geometry.Rotation2d;
-import frc.robot.constants.LogPaths;
-import frc.robot.subsystems.swerve.SwerveConstants;
 import frc.utils.Conversions;
 import frc.utils.battery.BatteryUtils;
 
-public record ModuleConstants(double wheelDiameterMeters, double couplingRatio, Rotation2d velocityAt12VoltsPerSecond) {
+public record ModuleConstants(
+        String logPath,
+        double wheelDiameterMeters,
+        double couplingRatio,
+        Rotation2d velocityAt12VoltsPerSecond
+) {
 
-    public ModuleConstants(double wheelDiameterMeters, double couplingRatio, double velocityAt12VoltsMetersPerSecond){
-        this(wheelDiameterMeters, couplingRatio, Conversions.distanceToAngle(velocityAt12VoltsMetersPerSecond, wheelDiameterMeters));
+    public ModuleConstants(
+            String swerveLogPath,
```

Comment:
Change to log path prefix or something like this, same in all relevant places

Reply:
I think prefix is less informative, "Subsystems/" is also a prefix to this log path...

Reply:
Here you get prefix to "modules/", you're right that most of the times it will be "swerve/" but its not a requirement.
The name swerveLogPath doesn't represent what the var mean but what to put in it